tox.ini

```diff
@@ -45,8 +45,8 @@ commands =
     ; coverage report.
     ;
     ; `--no-cov-on-fail` provide no code coverage on a failing test run.
-    py.test --cov-config={toxinidir}/.coveragerc --cov-append --cov-branch --no-cov-on-fail --cov=src/sub -k src/sub --ignore=src/sub/shared
-    py.test --cov-config={toxinidir}/.coveragerc --cov-append --cov-branch --no-cov-on-fail --cov=src/hub -k src/hub --ignore=src/hub/shared
-    py.test --cov-config={toxinidir}/.coveragerc --cov-append --cov-branch --no-cov-on-fail --cov=src/shared -k src/shared
+    py.test --cov-report term-missing --cov-config={toxinidir}/.coveragerc --cov-append --cov-branch --no-cov-on-fail --cov=src/sub -k src/sub --ignore=src/sub/shared
```

Comment:
Given that we have been doing discrete pull requests (PR), should we extract this change into it's own PR that is separate from this workstream?

Reply:
Good point. Done.